backend-rust/src/bin/ccdscan-api.rs

```diff
@@ -116,41 +116,32 @@ async fn main() -> anyhow::Result<()> {
     // Await for signal to shutdown or any of the tasks to stop.
     tokio::select! {
         _ = tokio::signal::ctrl_c() => {
-            info!("Received signal to shutdown");
```

Comment:
I think this info is useful, since it provides feedback to the user and since it is the graceful shutdown it might take some time